telegram/request/_baserequest.py

```diff
@@ -128,6 +128,23 @@ async def __aexit__(
         # https://docs.python.org/3/reference/datamodel.html?#object.__aexit__
         await self.shutdown()
 
+    @property
+    def read_timeout(self) -> Optional[float]:
+        """This property must return the default read timeout in seconds used by this class.
+        More precisely, the returned value should be the one used when
+        :paramref:`post.read_timeout` is not passed/equal to :attr:`DEFAULT_NONE`.
+
+        .. versionadded:: NEXT.VERSION
+        .. deprecated:: NEXT.VERSION
```

Comment:
makes sense to put this as a Caution since adding a "deprecation" note the same time as a "versionadded" doesn't quite make sense to me

Reply:
yeah, there is no ".. future" or something like that … change to "Warning".